Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM quay.io/operator-framework/ansible-operator:v1.3.0
+ARG ACC_PROVISION_REPO_BRANCH
+ENV ACC_PROVISION_BRANCH=${ACC_PROVISION_REPO_BRANCH:-master}
+
+COPY requirements.yml ${HOME}/requirements.yml
+USER 0
```

Comment:
do we require root user ?

Reply:
Tried without it but you must be root to run update-crypto-policies.

Reply:
ok sure.